main.sh

```diff
@@ -185,6 +185,21 @@ if [ -z "$OUTPUT_PATH" ]; then
   fi
 fi
 
+if ! command -v pciutils i2c-tools mesa-utils smartmontools dmidecode >/dev/null 2>&1; then
+  echo "There aren't all the dependencies needed. Do you want to install them? y/N"
```

Comment:
Make the fact that it won't work without the dependencies clearer
```suggestion
  echo "You need to install some packages in order for the peracotta to work. Do you want to install them? y/N"
```